packages/cli/src/types/handler.ts

```diff
@@ -0,0 +1,19 @@
+interface CMDHandlerObjectParams {
+  [key: string]: any;
```

Comment:
Is there possible to remove the any?

Reply:
any is better to handler:(

Reply:
See https://github.com/alibaba/pipcook/pull/140#discussion_r415523917.

Reply:
It is some special here :(

Reply:
What's the special case here?

Reply:
Ping @Txiaozhe

Reply:
resolved:)